File: launchdarkly/ruby/src/GetContextInstanceSegmentsMembershipByEnvExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::SegmentsApi.new.get_context_instance_segments_membership_by_env(
        "projectKey_string", # project_key
        "environmentKey_string", # environment_key
        JSON.parse(<<-EOD
            {
                "address": {
                    "city": "Springfield",
                    "street": "123 Main Street"
                },
                "jobFunction": "doctor",
                "key": "context-key-123abc",
                "kind": "user",
                "name": "Sandy"
            }
            EOD
        ), # request_body
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling SegmentsApi#get_context_instance_segments_membership_by_env: #{e}"
end

File: launchdarkly/typescript-node/src/GetContextInstanceSegmentsMembershipByEnvExample.ts
import * as fs from 'fs';
import api from "launchdarkly_client"
import models from "launchdarkly_client"

const apiCaller = new api.SegmentsApi();
apiCaller.setApiKey(api.SegmentsApiApiKeys.ApiKey, "YOUR_API_KEY");

apiCaller.getContextInstanceSegmentsMembershipByEnv(
  "projectKey_string", // projectKey
  "environmentKey_string", // environmentKey
  {
    "address": {
      "city": "Springfield",
      "street": "123 Main Street"
    },
    "jobFunction": "doctor",
    "key": "context-key-123abc",
    "kind": "user",
    "name": "Sandy"
  }, // requestBody
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling SegmentsApi#getContextInstanceSegmentsMembershipByEnv:");
  console.log(error.body);
});
